### PR TITLE
fix the thrift msgs installation

### PR DESCRIPTION
**Pull request by kouroshD**
Related issue:
https://github.com/dic-iit/element_retargeting-from-human/issues/169

cc @Yeshasvitvs  @lrapetti 
I have not tested the changes, but I could build it successfully.

### CODE REVIEW

**Review by diegoferigo — APPROVED** [review]
Changes look good and I'm blindly approving them. I'd wait someone testing this PR before merging.

**Review by traversaro — APPROVED** [review]

File: wrappers/HumanDynamicsWrapper/HumanDynamicsWrapper.cpp [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 #include "HumanDynamicsWrapper.h"
 #include <hde/interfaces/IHumanDynamics.h>
-#include <HumanDynamicsEstimation/HumanDynamics.h>
+#include <hde/msgs/HumanDynamics.h>>
```
Comment (lrapetti) [inline wrappers/HumanDynamicsWrapper/HumanDynamicsWrapper.cpp:11]:
```suggestion
#include <hde/msgs/HumanDynamics.h>
```

**Review by lrapetti — APPROVED** [review]
Thanks @kouroshD!

I have tested and everything works fine. Just one typo to be fixed

File: wrappers/HumanDynamicsWrapper/HumanDynamicsWrapper.cpp (hunk 1 above)
Comment (kouroshD) [reply wrappers/HumanDynamicsWrapper/HumanDynamicsWrapper.cpp]: @lrapetti  but with this typo, how it was building successfully!?

**Comment by lrapetti** [comment]
Thanks @kouroshD

**Pull request merged**